requirements.txt

```diff
@@ -1,3 +1,4 @@
+MarkupSafe==2.0.1
 canonicalwebteam.flask-base==0.9.3
```

Comment:
Can you move it below flask-base? we had issues before, whereas the module on top overrode the Python version of the rest of the modules. This way we ensure that the flask-base Python version is the one we use.